### PR TITLE
drive up test coverage, drop redundant integration test skip

### PR DESCRIPTION
**Pull request by BenTheElder**

### CODE REVIEW

File: cmd/archeio/main_test.go [hunk 1]
```diff
@@ -33,8 +33,6 @@ import (
 // TestIntegrationMain tests the entire, built binary with an integration
 // test, pulling images with crane
 func TestIntegrationMain(t *testing.T) {
-	integration.MaybeSkip(t)
```
Comment (BenTheElder) [inline cmd/archeio/main_test.go:36]: this was a hold-over from KIND where we iterated on this previously. the test tags and regex matching should be sufficient

File: pkg/net/cidrs/aws/internal/ranges2go/main.go [hunk 2]
```diff
@@ -27,11 +27,7 @@ func main() {
 		outputPath = "./zz_generated_range_data.go"
 	}
 	// parse raw AWS IP range data
-	parsed, err := parseIPRangesJSON([]byte(ipRangesRaw))
-	if err != nil {
-		panic(err)
-	}
-	rtp, err := regionsToPrefixesFromData(parsed)
+	rtp, err := regionsToPrefixesFromRaw(ipRangesRaw)
```
Comment (BenTheElder) [inline pkg/net/cidrs/aws/internal/ranges2go/main.go:30]: we want to minimize the amount of logic in any main.go, and ensure we're testing everything else thoroughly with quick and cheap unit tests ...

File: pkg/net/cidrs/aws/internal/ranges2go/parse_test.go [hunk 3]
```diff
@@ -77,3 +77,56 @@ func TestParseIPRangesJSON(t *testing.T) {
 		t.Fatal("expected error parsing garbage data but got none")
 	}
 }
+
+func TestRegionsToPrefixesFromData(t *testing.T) {
+	t.Run("bad IPv4 prefixes", func(t *testing.T) {
```
Comment (aojea) [inline pkg/net/cidrs/aws/internal/ranges2go/parse_test.go:82]: why not test valid prefixes too?  ... and use a table test, only expectError is tested

Comment (BenTheElder) [reply pkg/net/cidrs/aws/internal/ranges2go/parse_test.go:82]: there's other tests that cover this method, but we do not cover the error cases in those tests.

Comment (BenTheElder) [reply pkg/net/cidrs/aws/internal/ranges2go/parse_test.go:82]: table test seems excessive for N=2.

File: pkg/net/cidrs/aws/internal/ranges2go/main.go (hunk 2 above)
Comment (dims) [reply pkg/net/cidrs/aws/internal/ranges2go/main.go:30]: ack

File: pkg/net/cidrs/aws/internal/ranges2go/parse_test.go (hunk 3 above)
Comment (dims) [reply pkg/net/cidrs/aws/internal/ranges2go/parse_test.go:82]: ack

File: cmd/archeio/main_test.go (hunk 1 above)
Comment (dims) [reply cmd/archeio/main_test.go:36]: ack

**Pull request merged**